lti_consumer/templates/html/lti_iframe.html

```diff
@@ -1,8 +1,9 @@
+<%page args="launch_url" expression_filter="h"/>
```

Comment:
`initial_launch_url` sounds good. I have also added more detail about the modal launch to the comment in student.html.